stubs/site/source/_assets/css/main.css

```
@@ -0,0 +1,3 @@
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
```

```suggestion
@tailwind base;
@tailwind components;
@tailwind utilities;
```

This would let us remove the `postcss-import` plugin and dependency. Thoughts?

With `postcss-import` if you want to add custom CSS you _have_ to put it in another file and `@import` that file, this way you can add it to `main.css`.

---

@damiani and I spoke about this one earlier. Neither one of us really use postcss-import, but the thought was maybe it's make the transition from scans a little easier? I'm pretty neutral on including it or removing it.

---

Let's keep it in for now; I do think it will ease the transition for people used to using @imports from Sass. We can revisit at some point down the road.